### PR TITLE
Login feat

### PR DESCRIPTION
**Pull request by UrosPsd**

### CODE REVIEW

File: package-lock.json
Comment (Fickos) [inline package-lock.json:1]: Remove this file

File: src/components/AddSlotModal/AddSlotModal.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    console.log("Submited");
+    onClose();
+  };
+
+  if (!isOpen) {
+    return null;
+  }
+
+  return (
+    isOpen && (
```
Comment (Fickos) [inline src/components/AddSlotModal/AddSlotModal.tsx:25]: not clean code

File: src/components/AddSlotModal/AddSlotModal.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+const AddSlotsModal: React.FC<AddSlotsModalProps> = ({ isOpen, onClose }) => {
+  const [fromDate, setFromDate] = useState("");
+  const [toDate, setToDate] = useState("");
+  const [price, setPrice] = useState("");
+  const [weekendPrice, setWeekendPrice] = useState("");
+
+  const handleSubmit = () => {
+    console.log("Submited");
```
Comment (Fickos) [inline src/components/AddSlotModal/AddSlotModal.tsx:16]: remove console.log

File: src/components/Card/hotel.jpg
Comment (Fickos) [inline src/components/Card/hotel.jpg:1]: move image to assets folder

**Review by Fickos — APPROVED** [review]

**Review by svetaa99 — APPROVED** [review]

**Pull request merged**